### PR TITLE
Local image

### PR DESCRIPTION
**Pull request by b0r6**
#162

### CODE REVIEW

File: cmd/lxc-openvdc/main.go [hunk 1]
```diff
@@ -59,7 +59,6 @@ func main() {
 	imgPath = *_imgPath
 	cachePath := *_cachePath
 
-	lxcPath = "/usr/share/lxc/"
```
Comment (unakatsuo) [inline cmd/lxc-openvdc/main.go]: Here should be left as:

```
var lxcPath string
```

**Review by unakatsuo — CHANGES_REQUESTED** [review]

File: cmd/lxc-openvdc/main.go (hunk 1 above)
Comment (b0r6) [reply cmd/lxc-openvdc/main.go]: It's defined as that further up.

Comment (unakatsuo) [reply cmd/lxc-openvdc/main.go]: Sorry. I missed that.

**Review by unakatsuo — APPROVED** [review]

**Comment by Metallion** [comment]
Yay. \o/

**Comment by unakatsuo** [comment]
https://ci.openvdc.org/blue/rest/organizations/jenkins/pipelines/citest/branches/master/runs/188/nodes/36/log/

```
=== RUN   TestLocalImage
--- FAIL: TestLocalImage (61.55s)
	00_ssh.go:77: STDOUT:
	00_ssh.go:78: STDERR:
		i-0000000012 doesn't exist
	00_ssh.go:80: Running 'sudo lxc-info -n i-0000000012' on '10.0.100.14' failed. Tried 10 times.
```

File: ci/citest/acceptance-test/tests/local_image_test.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import (
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestLocalImage(t *testing.T) {
+
+	// Use custom lxc-template.
+	stdout, _ := RunCmdAndReportFail(t, "openvdc", "run", "centos/7/lxc", `{"lxc_template":{"template":"openvdc"}}`)
```
Comment (unakatsuo) [inline ci/citest/acceptance-test/tests/local_image_test.go:14]: Here is missing ``node_groups`` attribute to let the instance run on ``executor_lxc_ip`` 10.100.0.14.

```
`{"lxc_template":{"template":"openvdc"}, "node_groups":"linuxbr"}`
```

**Pull request merged**